### PR TITLE
Correcting and improving translation

### PR DESCRIPTION
**Pull request by maiconkkl**

### CODE REVIEW

File: locales/br.lua [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  ['started_to_rob'] = 'você começou a roubar ~y~%s~s~',
+  ['alarm_triggered'] = 'o alarme foi acionado',
+  ['robbery_complete'] = '~r~O assalto foi bem sucedido~s~, você ~o~roubou~s~ ~g~$%s~s~',
+  ['robbery_complete_at'] = '~r~Assalto bem sucedido em ~y~%s~s~',
+  ['robbery_cancelled'] = 'o assalto foi cancelado!',
+  ['robbery_cancelled_at'] = '~r~O assalto em ~b~%s~s~ foi cancelado!',
+  ['min_police'] = 'deve haver pelo menos ~b~%s policiais~s~ na cidade para roubar.',
+  ['robbery_already'] = '~r~Um assalto já está em andamento.',
+  ['no_threat'] = 'você não representa uma ameaça para o lojista',
+}
```
Comment (HawaiiBeach) [inline locales/br.lua:16]: trailing missing

**Review by ElPumpo — CHANGES_REQUESTED** [review]

File: locales/br.lua (hunk 1 above)
Comment (maiconkkl) [reply locales/br.lua:16]: fixed

**Review by maiconkkl — COMMENTED** [review]
fixed

**Comment by kibonara** [comment]
requests added, thanks for ur help

**Comment by HawaiiBeach** [comment]
@kibonara that's NOT how you accept a pull request?!

**Pull request closed without merging**